app/Console/Commands/RebuildImages.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Poniverse\Ponyfm\Console\Commands;
+
+use Illuminate\Console\Command;
+use Poniverse\Ponyfm\Models\Image;
+use Symfony\Component\HttpFoundation\File\File;
+
+class RebuildImages extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rebuild:images';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Resizes all images to fit the specifications in Models/Image';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $images = Image::all();
+        foreach ($images as $image) {
+            $this->info("Regenerating images for id:".$image->id. " (".$image->filename.")");
+            $image->clearExisting();
+
+            $originalFile = new File($image->getFile(Image::ORIGINAL));
+            foreach (Image::$ImageTypes as $imageType) {
+                Image::processFile($originalFile, $image->getFile($imageType['id']), $imageType);
```

Comment:
What happens if this function or the shell call inside it fail? CLI tasks that are prone to failure (and a task like this one which touches thousands of files is) should have rudimentary error handling to make debugging easy if and when something goes wrong.

One way to accomplish that here is to wrap this call in a try-catch, log info about the exception and the image it was trying to process if the call fails, and make sure that the `processFile` function itself will throw a descriptive exception when something goes wrong.